CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Changelog
 
 0.3.6
+
+* Added support to debug a site without appending the debug parameters in the URL
```

Comment:
Actually we can remove this changelog file @AltamashShaikh these changes will be mentioned instead in Matomo On-Premise changelog

Reply:
removed